__tests__/app.test.js

```diff
@@ -77,6 +77,64 @@ describe("/api", () => {
             });
           });
       });
+      test("Sort order should default to date", () => {
+        return request(app)
+          .get("/api/reviews")
+          .expect(200)
+          .then(({ body: { reviews } }) => {
+            const dates = reviews.map((review) => new Date(review.created_at)),
+              sortedDates = [...dates].sort((a, b) => b - a);
```

Comment:
This is a nice example of problem solving, but you'd make your life a lot simpler (and your code a lot easier to read) if you were to look at using jest-sorted here.

Reply:
It would also make it simpler to check whether the order is descending in one assertion.